README.md

```diff
@@ -150,8 +149,7 @@ This library should only be used with the Okta management API. To call the [Auth
 
 #### Before each Example
 
-Assume the client is instantiated before each example below.
-
+> Assume the client is instantiated before each example below.
 ```py
 from okta.client import Client as OktaClient
 import okta.models as models
```

Comment:
Sorry... I would change this to be the following, and just remove the header all together so it lives in the `h2` above of Examples

--------------

> Each of the following examples assumes the client is instantiated as below:
> ```py
> from okta.client import Client as OktaClient
> import okta.models as models
> ```